docs/graph2d/index.html

```diff
@@ -557,6 +557,36 @@ <h3 id="graph2dOptions">Graph2d Options</h3>
             <td>Toggle the drawing of the datapoints.</td>
         </tr>
         <tr parent="drawPoints" class="hidden">
+            <td class="greenField indent">drawPoints.onRender</td>
+            <td>function</td>
+            <td>none</td>
+            <td>Defines a render option for every datapoint. Will only be invoked when <code>drawPoints.enabled</code> is set to <code>true</code>.
+            When a group has no <code>drawPoints.onRender</code> callback, the graph2d <code>drawPointsonRender</code> callback will be used. If neither
+            is defined, the datapoint will be rendered according to the group setting of <code>drawPoints.enabled</code>.
+            This callback must return <code>true</code> if the datapoint should be rendered, otherwise <code>false</code>.
+              <pre class="prettyprint lang-js">
+drawPoints: {
+  enable: true,
```

Comment:
this should be enableD I think :). You do not have to add the enabled though, it is set to true whenever you pass a property inside of the object